### DISCUSSION TITLE
Upgrading on Windows binary

### DISCUSSION THREAD

**Discussion by Mysticodex** [Eloston/ungoogled-chromium#1411] [Q&A]
With the Windows binary installed, how do I do an upgrade of the current version of Ungoogled Chromium?

**Comment by tangalbert919** [comment]
Just download an updated version of the installer and run it.

**Comment by Mysticodex** [comment]
Yes, this has worked. When I ran the installer, it didn't show any progress
bar, installer UI, etc. It just opened and closed immediately, and the app
was updated. This is why I was confused about how to upgrade.
